submissions/RomanSaiko/dom-practise/js/scripts.js

```diff
@@ -0,0 +1,80 @@
+const content = [
+    {
+        id: '0',
+        city: 'Paris',
+        country: 'France',
+        region: 'Île-de-France',
+        population: '2,148,271',
+        area: '105.4',
+        website: 'https://en.parisinfo.com/'
+    },
+    {
+        id: '1',
+        city: 'London',
+        country: 'England',
+        region: 'London',
+        population: '8,961,989',
+        area: '1,572',
+        website: 'https://www.london.gov.uk/'
+    },
+    {
+        id: '2',
+        city: 'Rome',
+        country: 'Italy',
+        region: 'Lazio',
+        population: '2,837,332',
+        area: '1,285',
+        website: 'https://www.rome.net/'
+    },
+    {
+        id: '3',
+        city: 'Amsterdam',
+        country: 'Netherlands',
+        region: 'North Holland',
+        population: '872,680',
+        area: '219.32',
+        website: 'https://www.amsterdam.nl/en/'
+    }
+];
+
+let renderPage = () => {
+    const iconMenu = document.getElementById('menu-handler');
+    const sidebarMenu = document.getElementById('aside-menu');
+    const article = document.getElementById('article');
+    const navMenu = document.getElementById('nav-list');
+
+    const handleMenuToggle = () => {
+        iconMenu.classList.toggle('open');
+        if (iconMenu.classList.contains('open')) {
+            sidebarMenu.classList.add('open');
+        }
+        else {
+            sidebarMenu.classList.remove('open');
+        }
+    };
+
+    const changeInfo = (e) => {
```

Comment:
You can use [destructuring](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Operators/%D0%94%D0%B5%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D1%96%D1%8F)

Reply:
Hi, are you talking about destructuring for lines 64-69?

Reply:
Something like this
```
const changeInfo = ({target}) => {
    const id = target.dataset.id;
```